build/dependencies.yaml

```diff
@@ -75,6 +75,8 @@ dependencies:
       match: quay.io/coreos/etcd
     - path: test/e2e/framework/nodes_util.go
       match: const etcdImage
+    - path: test/utils/image/manifest.go
+      match: configs\[Etcd\] = Config{list\.GcEtcdRegistry, "etcd", "\d+\.\d+.\d+(-(alpha|beta|rc).\d+)?(-\d+)?"}
 
   - name: "etcd-image"
```

Comment:
leaving a breadcrumb (https://kubernetes.slack.com/archives/C09QZ4DQB/p1637362487110300), we need to move this down to the ectd-image group at some point (my bad!)